README.md

```diff
@@ -20,12 +20,12 @@ Medical annotation tool
 
 # Run
 
-* `cd api; FLASK_ENV="development" flask run`
+* `cd api; uvicorn main:app --reload`
```

Comment:
I think the command is `uvicorn mod.src.app:APP --reload` or sth like that, right?

Reply:
Already fixed with the latest commit :)